frontend/device/generic/powerd.lua

```diff
@@ -17,6 +16,7 @@ local BasePowerD = {
     last_aux_capacity_pull_time = time.s(-61),  -- timestamp of last pull
 
     is_fl_on = false,                 -- whether the frontlight is on
+    UIManager = nil -- will be updated when available
```

Comment:
I'm a bit torn on that one ;).

I understand the reasoning, but I'm not necessarily a fan of it.

If we can't think of anything better, I would at least rename it to `ui_mgr` to make it clear it's not our usual module-local.

(There's no real technical issues, though, given that UIManager is a singleton. The testsuite might rain on your parade, though).